src/main/webapp/index.html

```diff
@@ -11,5 +11,6 @@
           document.querySelector('p').innerText = text;
       });
   </script>
+  <a href="/auth">Login | Logout</a>
```

Comment:
It's better to add all tags in `<body>` before `<script>` tag, because if your js file will need some DOM element after `<script>` tag, it can not to find it.